README.md

```diff
@@ -0,0 +1,34 @@
+Jamal
+=====
+
+YAML to JSON and JSON to YAML conversion tool. Written in Golang so it can be statically compiled and dropped anywhere you need it.
+
+[Build of latest release.](https://github.com/quantumew/jamal/releases)
+
+
+Usage
+-----
+
+    ./jamal <action> [<input-file>]
+
+    Options:
+        -h --help       Show this message.
+
+    Arguments:
+        <action>        Conversion action. [yaml2json|json2yaml]
```

Comment:
This documentation doesn't have the `yamltojson` and `jsontoyaml` options. That said, I don't think they're really necessary and I'd prefer to have briefer`y2j` or `j2y` options.